torch/_export/serde/serialize.py

```diff
@@ -465,12 +467,11 @@ def serialize_metadata(self, node: torch.fx.Node) -> Dict[str, str]:
                 f"{k}:({v[0]},{self.serialize_operator(v[1])})"
                 for k, v in nn_module_stack.items()
             ]
-            ret["nn_module_stack"] = ";".join(nn_module_list)
+            ret["nn_module_stack"] = ST_DELIMITER.join(nn_module_list)
```

Comment:
since we are technically changing the IR, does ExportedProgram's version number need a bump? Or is the version number bump automatic (and in-line with the PyTorch version number?)